### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by sbstn87** [Nheko-Reborn/nheko#1736]
### The Problem

The last release has been don in February 2023. It is such along time ago that it is available int he repos of Debian Stable.

### The Solution

Make a new release to avoid a misconception that the project is stale.

### Alternatives

Make downloading of nightly releases more visible.

### Happens in the latest version

- [X] Yes, this feature is still missing.

**Comment by sladiri** [comment]
Hi, I was just stumbling upon the Flatpak warning, that Nheko does not receive updates anymore.
I added the nightly repo from `https://github.com/Nheko-Reborn/nheko/blob/master/nheko-nightly.flatpakrepo` but the version seems to be the same. Is the Flatpak nightly not providing the latest version?

**Comment by XDMelone333** [comment]
I have to say, i strongly agree with this. I love Nheko, but it hasn't recieved updates on flathub for over a year... just why?
i could find absolutely no reasoning for it anywhere on the github page and also no other instructions to install a more up to date version.

I'd love to keep using nheko, but i won't like this.

**Comment by deepbluev7** [comment]
There has been no release, because a release takes effort and I didn't have time for that recently. We need to change our flathub packaging, because their verification system doesn't work with out old flatpak id and I didn't have motivation to do that yet. Also not shipping a release for a year is perfectly normal, not everything needs weekly releases and since we migrated to Qt6, we wanted to take the time we needed to squash regressions caused by that. This was already answered multiple times and the next time someone files an issue asking for a release, it will get closed without a comment. Use the search feature Github has, that would save a lot of time for me and might actually give me the time to push out a release instead of answering the same questions over and over. :)

**Comment by sladiri** [comment]
I try to search for issues but in this case I guess I felt rushed by the red Flatpak warning text. When I saw this ticket, I somehow overlooked the closed issues and assumed this was to track the progress of how to deal with the Flatpak warning. Love using nheko!

**Comment by Mikaela** [comment]
Note that the nightly flatpak id changed some time ago.

I guess I can link this since it's so commonly linked in Matrix as well

- https://aminda.eu/matrix/#how-can-i-install-nheko-nightly

**Comment by sladiri** [comment]
Thank you @Mikaela, I have installed the nightly! I tried to install the nightly before as I said but that failed I think. This is also why I assumed that there were some recent changes about it and asked here.

**Comment by LorenDB** [comment]
@sladiri FYI, we just released 0.12.0, so you should be able to install that now.

**Comment by NintendoManiac64** [comment]
Unfortunately 0.12.0 includes the pretty bad usability bug of issue #1752.

Now I understand that, if you use Nheko/matrix more akin to IRC, then this probably isn't that big of a deal. But I used it more as a modern replacement for instant messaging (e.g. ICQ and such), so not having the notification dot means I will straight-up miss that there's a new message at all if it's sent when I'm away-from-keyboard (which *did* happen recently).

While issue #1752 has since been fixed, the problem is that, if history is anything to go by, we could be waiting another 18 months until a new version gets released that includes that fix. Worse still is that, from my knowledge, flathub doesn't store older versions. **10 month later EDIT**: Turns out flathub *does* have the older versions, but this was under the old name "nhekoReborn" and I'm not enough of a Linux guru to know how do undo that name-change situation... but I've since been making due with the AppImage version of 0.11.3 anyway.

　

(there are various work-arounds to make sure I don't receive notifications while AFK, but they all involve either compromise, e.g. I can't have something downloading while AFK, and/or ideally take advantage of hardware and/or functions that are being discontinued such as s3 suspend, because s2idle retains network connectivity, or Optane memory which has much more write endurance making it suitable for frequent use of hibernation on hardware lacking s3 support)
